### PR TITLE
Update API test for retrieving app config.

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

**Review by DwayneJengSage — COMMENTED** [review]
How easy is it to test the AppConfig API publicly (without a user)? If it's easy to do, we should change the test to do that.

If not, this is probably fine.

**Comment by alxdarksage** [comment]
Good point. I changed the end of the test to use the public API and verified that it is filtering.

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/AppConfigTest.java [hunk 1]
```diff
@@ -33,27 +36,21 @@
 
 public class AppConfigTest {
     
-    private TestUser user;
+    //private TestUser user;
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/AppConfigTest.java]: Minor: Remove commented out user. (And equivalent code in before() and after().

**Review by DwayneJengSage — APPROVED** [review]
One minor comment. Otherwise, looks good.

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**